dune_client/models.py

```diff
@@ -354,3 +354,24 @@ def __add__(self, other: ResultsResponse) -> ResultsResponse:
         self.next_uri = other.next_uri
         self.next_offset = other.next_offset
         return self
+
+
+@dataclass
+class CreateTableResponse(DataClassJsonMixin):
+    """
+    Data type returned by table/create operation
+    """
+
+    example_query: str
+    full_name: str
+    namespace: str
+    table_name: str
+
+
+@dataclass
+class InsertTableResult(DataClassJsonMixin):
```

Comment:
Can we change this to `InsertTableResponse` for consistency with `CreateTableResponse`?

Reply:
Or how about both with "..Result"